README.md

```diff
@@ -25,21 +25,45 @@ This starter uses [Remix](https://remix.run/) for the frontend and [Sanity](http
 
 The following commands are meant to be run in **both** the `/app` and `/studio` folders.
```

Comment:
Is this line still relevant?

i.e. `pnpm install` will install for all the workspace projects, and `pnpm dev` script is recursive.